pkg/connection/prepared_stmt_converter.go

```diff
@@ -0,0 +1,93 @@
+package connection
+
+import (
+	"database/sql/driver"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/exasol/exasol-driver-go/pkg/errors"
+	"github.com/exasol/exasol-driver-go/pkg/types"
+)
+
+func convertArg(arg driver.Value, colType types.SqlQueryColumnType) (interface{}, error) {
+	dataType := colType.Type
+	switch dataType {
+	case "DOUBLE":
+		switch arg := arg.(type) {
```

Comment:
If I'd had to take a guess this is probably where the quality gate failed.

Reply:
You probably need this: https://stackoverflow.com/questions/68578/multiple-cases-in-switch-statement